app/javascript/controllers/tinymce_controller.js

```diff
@@ -0,0 +1,30 @@
+// controller for tinymce
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  connect() {
+    tinymce.init({
+        selector: ".tinymce",
+        plugins: [
+          "image",
+          "advlist autolink lists link image charmap print preview anchor",
+          "searchreplace visualblocks code fullscreen",
+          "insertdatetime media table paste code help wordcount",
+        ],
+        toolbar:
+          "file edit view insert format tools undo redo | formatselect | bold italic backcolor | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | removeformat | help | image",
+        content_style: "body {font-size: 11pt; color: #374151}",
+        skin: "oxide-dark",
+
+      });
+  }
+  
+  // disconnect() {
+  //  // tinymce.activeEditor.destroy()
+  // }
```

Comment:
Is this the recommended implementation?

Reply:
Thanks for highlighting that Jake, I've changed it.